bib/cmd/bootc-image-builder/main.go

```diff
@@ -422,6 +432,8 @@ func run() error {
 	manifestCmd.Flags().String("rpmmd", "/rpmmd", "rpm metadata cache directory")
 	manifestCmd.Flags().String("target-arch", "", "build for the given target architecture (experimental)")
 	manifestCmd.Flags().StringArray("type", []string{"qcow2"}, "image types to build [qcow2, ami, iso, raw]")
+	manifestCmd.Flags().Bool("remote", false, "use a container image from a remote registry")
```

Comment:
@ondrejbudai made a good point, we could maybe remove this flag entirely and just have the single codepath?